spec/valkyrie/persistence/fedora/metadata_adapter_spec.rb

```diff
@@ -22,12 +28,16 @@
 
       describe "#id_to_uri" do
         it "converts ids with a slash" do
-          id = "test/default"
+          id = "test1234/default"
           if adapter.fedora_version == 4
 
+            id = "test/default"
             expect(adapter.id_to_uri(id).to_s).to eq "http://localhost:8988/rest/test_fed/te/st/test%2Fdefault"
```

Comment:
What do you think about changing the test case to match the other cases?
```suggestion
            expect(adapter.id_to_uri(id).to_s).to eq "http://localhost:8988/rest/test_fed/te/st/12/34/test1234%2Fdefault"
```